core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -186,6 +186,9 @@ object ConsumerPerformance extends LazyLogging {
           lastBytesRead = bytesRead
         }
       }
+      if (messagesRead < count)
+        println(s"WARNING: Exiting before consuming the expected number of messages: timeout ($timeout ms) exceeded. " +
+          s"Probably too much time spent on processing a polled list of records. Be sure to increase the --timeout option")
```

Comment:
Nit: Missing "." at the end.

Reply:
I also wonder if this is the right place to print this message. Perhaps this should be outside the `while` loop?

Reply:
@dhruvilshah3, my bad, thank you for pointing this out. Updated